lib/tests/httpSessionExist.test.js

```diff
@@ -0,0 +1,45 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import isUserSessionExist from "../middleware/HttpSessionExist";
+
+jest.mock("next-auth/client");
+
+const handler = async (req, res) => {
```

Comment:
This should be a mock function. That way you can test if its actually called ! But in any case this works for this purpose just for future reference